src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,8 +9,9 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
-    public static final String VALIDATION_REGEX = "\\p{Alnum}+";
+    public static final String MESSAGE_CONSTRAINTS = "Tags should not be blank, and"
+            + "should be at most 30 characters";
```

Comment:
Is there a missing space?

_Current Display:_
![image](https://user-images.githubusercontent.com/86542359/224657670-81756c9a-3690-4065-8c7c-1ef7e961d110.png)